addon/converter/fixture-converter.js

```diff
@@ -99,8 +99,8 @@ export default class FixtureConverter {
   }
 
   transformRelationshipKey(relationship) {
-    let {parentType} = relationship,
-        type         = parentType && parentType.modelName || relationship.type,
+    let {parentModelName, parentType} = relationship,
```

Comment:
Is `parentType` still a property `relationship`? Possibly worth keeping this as a fallback for backwards compatibility if it was.

Reply:
your right, good to keep for backward compat,  ( not absolutely necessary because I make sure every version 3.8 , etc works with 3.8 ED ) but just a nice to have.